src/networking/backend.ts

```diff
@@ -200,4 +201,41 @@ export class Backend {
       },
     );
   }
+
+  async setAttributes(
```

Comment:
One thing that I was considering, in the mobile SDKs, since we cache these, we basically don't send them again if the value hasn't changed, which can potentially save on requests. I can see how some devs might be calling this every time, and we would set the attributes every time. I think this is probably fine for now, but just wanted to call it out so we're aware.